hyperspy/misc/eds/utils.py

```diff
@@ -243,6 +275,74 @@ def take_off_angle(tilt_stage,
                                   + math.sin(a) * math.sin(c)))
 
 
+def xray_lines_model(elements,
+                     beam_energy=200,
+                     weight_percents=None,
+                     energy_resolution_MnKa=130,
+                     energy_axis={'name': 'E', 'scale': 0.01, 'units': 'keV',
+                                  'offset': -0.1, 'size': 1024}
```

Comment:
Should not use mutable types as default values.
Instead leave default value as `None`, and if a better thing is required, just create the dict in the first couple of lines of the actual code:
```python
if energy_axis is None:
    energy_axis = {'name': 'E', 'scale': 0.01, 'units': 'keV',
                   'offset': -0.1, 'size': 1024}
```